conda_env/utils/binstar_uilts.py

```diff
@@ -0,0 +1,107 @@
+from binstar_client import errors
```

Comment:
Typo in the file name.  Should also probably just be `utils/binstar.py` to avoid being redundant.